src/common/utils.js

```diff
@@ -113,11 +117,15 @@ function isRoll20(title) {
 }
 
 function isFVTT(title) {
-    return title.includes("Foundry Virtual Tabletop");
+    return title.includes("Foundry Virtual Tabletop") || title.includes("Foundry VTT");
 }
 
 function fvttTitle(title) {
-    return title.replace(" • Foundry Virtual Tabletop", "");
+    return title
```

Comment:
This change shouldn't be necessary since Beyond20 is the one that sets the title and forces it to have " • Foundry Virtual Tabletop"

See https://github.com/sirrio/Beyond20/blob/641629eb130bf6ff33f2b1a0a9f30950ee68457e/src/fvtt/page-script.js#L573-L577

Having that string in it means that Beyond20 is properly loaded on it and the chat UI is rendered so it can receive rolls.